code/modules/recycling/conveyor2.dm

```diff
@@ -55,6 +56,7 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 	if(newid)
 		id = newid
 	update_move_direction()
+	RegisterSignal(loc, list(COMSIG_ATOM_ENTERED, COMSIG_ATOM_CREATED), .proc/conveyorCrossed) // When any atom moves onto the belt or gets created over the belt.
```

Comment:
Think you could add an `if(operating)` check, and register when turning on and unregister when turning off?

Reply:
Yeah I can do that, will be more efficient probably.